apheleia-formatters.el

```diff
@@ -968,6 +971,13 @@ cmd is to be run."
       (setf (apheleia-formatter--name context) name)
       (setf (apheleia-formatter--stdin context) stdin)
       (setf (apheleia-formatter--remote context) remote)
+
+      ;; Just use npx when buffer is remote or we're running on Windows,
+      ;; since in these cases bash won't be able to find the script
+      (when (and (member "apheleia-npx" command)
+                 (or run-on-remote (eq system-type 'windows-nt)))
+        (setq command (cons 'npx (remove "apheleia-npx" command))))
```

Comment:
This will be an issue for any of the scripts in the Apheleia scripts directory, just just the npx one. What would you think of having Apheleia copy the script to the remote host before first execution, if anything `apheleia`-prefixed is used in the command?